package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@internetarchive/your-webcomponent",
-  "version": "0.0.0",
+  "name": "@internetarchive/throttler-debouncer",
+  "version": "0.0.1-alpha.1",
```

Comment:
For versioning, you typically don't want to bump the versions in functional pull requests. Generally, the idea is that you would have pull requests accumulate into your `main` branch, then once you're ready to do a release, bump the version in the main branch and release all of the code at once. This keeps your `main` branch at the currently released version.

Reply:
updated